code/modules/projectiles/ammo_datums.dm

```diff
@@ -1358,6 +1358,43 @@ GLOBAL_LIST_INIT(no_sticky_resin, typecacheof(list(/obj/item/clothing/mask/faceh
 	accuracy_var_low = 3
 	accuracy_var_high = 3
 
+/datum/ammo/energy/plasma_pistol
+	name = "ionized plasma bolt"
+	icon_state = "overchargedlaser"
+	hud_state = "laser_sniper"
+	damage = 20
+	max_range = 40
+	penetration = 5
+	sundering = 0
+
+	flags_ammo_behavior = AMMO_ENERGY|AMMO_INCENDIARY|AMMO_SUNDERING|AMMO_EXPLOSIVE
+
+	bullet_color = COLOR_VIBRANT_LIME
+
+	///Fire burn time
+	var/heat = 12 
+	///Fire damage
+	var/burn = 6
+	///Fire color
+	var/f_color = "green" 
+
+/datum/ammo/energy/plasma_pistol/on_hit_turf(turf/T, obj/projectile/proj)
+	if(!istype(T))
+		return
```

Comment:
Why?

Reply:
cuz that was in the nuero spit code that i copied a bit from

Reply:
Bad